packages/test/src/test-integration-workflows.ts

```diff
@@ -1,7 +1,8 @@
 import { randomUUID } from 'crypto';
 import { ExecutionContext } from 'ava';
 import { firstValueFrom, Subject } from 'rxjs';
-import { WorkflowFailedError } from '@temporalio/client';
+import asyncRetry from 'async-retry';
```

Comment:
There are some lint warnings here. Make sure to execute `npm run lint` before pushing to GitHub.

Reply:
I'd run this prior to pushing, does it not make code changes, just warnings?
In any case, I've addressed the warnings.